### PR TITLE
Projects redesign/prevent sudden zoom

### PR DESCRIPTION
**Pull request by sunilsabatp**
- [x] -  Zoom to center of results on search/filter??
- [x] -  Check zoom behavior for zooming into site, should stop before last "jump"

### CODE REVIEW

**Comment by mohitb35** [comment]
Functionality works perfectly, good work @sunilsabatp

**Review by mariahosfeld — COMMENTED** [review]
Functionality looks good to me

File: src/features/projectsV2/ProjectsMap/index.tsx
Comment (mohitb35) [inline src/features/projectsV2/ProjectsMap/index.tsx:5]: Clean up unwanted deps

File: src/utils/projectV2.ts
Comment (mohitb35) [inline src/utils/projectV2.ts:17]: MapRef needs to be imported from /types/projectv2

File: src/utils/projectV2.ts
Comment (mohitb35) [inline src/utils/projectV2.ts:195]: Resolve the TS warning here

**Review by mohitb35 — APPROVED** [review]
@sunilsabatp Minor TS warnings to be addressed. LGTM after that's done, no need for another review.

**Pull request merged**
